bosh-release/README.md

```diff
@@ -1,127 +1,43 @@
-
-      ```
-      -  config/repository/components/lib/mysql-connector-java-5.1.36.jar
-      ```
-
-    v.  Provide your svn configuration credentials in wso2-is-bosh-release/src/config/repository/conf/carbon.xml for deployment synchronization
-
-    ```
-    <DeploymentSynchronizer>
-        <Enabled>true</Enabled>
-        <AutoCommit>true</AutoCommit>
-        <AutoCheckout>true</AutoCheckout>
-        <RepositoryType>svn</RepositoryType>
-        <SvnUrl>https://svn.riouxsvn.com/wso2is</SvnUrl>
-        <SvnUser>username</SvnUser>
-        <SvnPassword>password</SvnPassword>
-        <SvnUrlAppendTenantId>true</SvnUrlAppendTenantId>
-    </DeploymentSynchronizer>```
-  
-
-3. Then get configuration files that specify BOSH environment in VirtualBox and run bosh create-env as following:
-
-    ```
-    $ git clone https://github.com/cloudfoundry/bosh-deployment bosh-deployment
-    $ mkdir vbox
-    $ bosh create-env bosh-deployment/bosh.yml \
-    --state vbox/state.json \
-    -o bosh-deployment/virtualbox/cpi.yml \
-    -o bosh-deployment/virtualbox/outbound-network.yml \
-    -o bosh-deployment/bosh-lite.yml \
-    -o bosh-deployment/bosh-lite-runc.yml \
-    -o bosh-deployment/jumpbox-user.yml \
-    --vars-store vbox/creds.yml \
-    -v director_name="Bosh Lite Director" \
-    -v internal_ip=192.168.50.6 \
-    -v internal_gw=192.168.50.1 \
-    -v internal_cidr=192.168.50.0/24 \
-    -v outbound_network_name=NatNetwork
-    ```
-
-4. Once VM with BOSH Director is running, point your CLI to it, saving the environment with the alias vbox:
-
-    ```
-    bosh -e 192.168.50.6 alias-env vbox --ca-cert <(bosh int vbox/creds.yml --path /director_ssl/ca)
-    ```
-
-5. Obtain generated password to BOSH Director:
-
-    ```
-    bosh int vbox/creds.yml --path /admin_password
-    ```
-
-6. Log in using admin username and generated password:
-
-    ```
-    bosh -e vbox login
-    ```
-
-7. Download Oracle JDK 1.8 from Oracle website and WSO2 Identity Server 5.3.0 via WSO2 Update Manager (WUM).
+## Prerequisites
 
-8. Add above distributions as blobs:
+1. Linux/Unix environment
+2. Install [BOSH Lite](https://bosh.io/docs/bosh-lite.html#install)
+3. Install [Docker](https://docs.docker.com/engine/installation/)
+4. (Optional)Install [WSO2 Update Manager](http://wso2.com/wum)
 
-    ```
-    bosh -e vbox add-blob jdk-8u144-linux-x64.tar.gz oraclejdk/jdk-8u144-linux-x64.tar.gz
-    bosh -e vbox add-blob wso2is-5.3.0.zip wso2is/wso2is-5.3.0.zip
-    bosh -e vbox -n upload-blobs
-    ```
+## Quick Start Guide
 
-9. Create the WSO2 Identity Server bosh release:
+1. Create a workspace directory
+   ```
+   mkdir wso2-is-bosh-workspace
+   ```
 
+2. Navigate to the workspace directory and clone this repository
     ```
-    bosh -e vbox create-release --force
+    cd wso2-is-bosh-workspace
+    git clone https://github.com/wso2/pivotal-cf-is --depth 1
```

Comment:
Why do we specify the depth as one here?

Reply:
It's just to pull up a shallow copy without the history.
But may be it looks confusing here as specifically instructed. Will remove